pss/pss.go

```diff
@@ -495,6 +492,7 @@ func (p *Pss) handlePssMsg(ctx context.Context, pssmsg *message.Message) error {
 // Attempts symmetric and asymmetric decryption with stored keys.
 // Dispatches message to all handlers matching the message topic
 func (p *Pss) process(pssmsg *message.Message, raw bool, prox bool) error {
+	metrics.GetOrRegisterCounter("pss.process.num", nil).Inc(1)
 	defer metrics.GetOrRegisterResettingTimer("pss.process", nil).UpdateSince(time.Now())
 
 	var err error
```

Comment:
This error variable is no longer needed.

Reply:
It is needed but I have moved it closer to where is declared and return nil at the end of the function.